cells/klayout/pymacros/cells/draw_mos.py

```diff
@@ -964,12 +964,12 @@ def draw_nmos_6p0_nat(layout, l, w, ld, nf, grw, bulk):
                 pya.Point(-(nat_enc_cmp + cmp2cmp) - grw, (w + (nat_enc_cmp + cmp2cmp))),
                 pya.Point(-(nat_enc_cmp + cmp2cmp), (w + (nat_enc_cmp + cmp2cmp))),
                 pya.Point(-(nat_enc_cmp + cmp2cmp), -(nat_enc_cmp + cmp2cmp)),
-                pya.Point((2 * ld + l + (nat_enc_cmp + cmp2cmp) + (nf - 1) * (ld + l)), -(nat_enc_cmp + cmp2cmp)),
-                pya.Point((2 * ld + l + (nat_enc_cmp + cmp2cmp) + (nf - 1) * (ld + l)), (w + (nat_enc_cmp + cmp2cmp))),
+                pya.Point((2 * ld + l + (nat_enc_cmp + cmp2cmp) + (nf - 1) * (ld + l + cont2ply - cmp2cont)), -(nat_enc_cmp + cmp2cmp)),
```

Comment:
refactor(ld + l + cont2ply - cmp2cont) into a variable. It will prevent errors when modifying the value in multiple places